discovery-installer.spec

```diff
@@ -0,0 +1,56 @@
+Name:           discovery-installer
+Version:        1.7
+Release:        1%{?dist}
+Summary:        Discovery Application v1.7 Installer
+
+License:        GPLv3
+URL:            https://gihub.com/quipucords/discovery-quadlets
+Source0:        %{name}-%{version}.tar.gz
+
+BuildArch:      noarch
+
+Requires:       bash
+
+%description
+This RPM installs the Discovery Application v1.7 Installer on the System.
+The Installer installs and configures Discovery v1.7 to be managed 
+and run via systemd quadlet services.
+
+%prep
+%autosetup -n %{name}-%{version}
+
+%install
+mkdir -p %{buildroot}/%{_bindir}
+mkdir -p %{buildroot}/etc/%{name}/bin
+cp bin/%{name} %{buildroot}/%{_bindir}/%{name}
+cp bin/create-server-password %{buildroot}/etc/%{name}/bin/
+cp bin/create-app-secret %{buildroot}/etc/%{name}/bin/
+cp bin/open-firewall-port %{buildroot}/etc/%{name}/bin/
+mkdir -p %{buildroot}/etc/%{name}/config
+cp config/*.container config/*.network %{buildroot}/etc/%{name}/config/
+mkdir -p %{buildroot}/etc/%{name}/env
+cp config/*.env %{buildroot}/etc/%{name}/env/
+
+%files
+%license LICENSE
+%doc README.md
+%{_bindir}/%{name}
+/etc/%{name}/bin/create-server-password
```

Comment:
it seems a bit odd to me to choose /etc instead of /usr or /opt.

how about we use `%{_datadir}/%{name}` instead of `/etc/%{name}`?

Reply:
I was thinking of system wide services (/etc, aka %{_sysconfdir}), but you're right since we're not installing as root or part of the OS, %{_datadir} is good. 👍